Jarvis-Platform/jarvis-studio
# Feature/generic table to storage

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

 [![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Use generic components for `table-to-storage` views


## Todos
### Runs
- [x] Use `listing-component` for `ListingView`
- [x] Use `item-component` for `ItemView`

### Configurations
- [x] Use `listing-component` for `ListingView`
- [x] Use `item-component` for `ItemView`


## Deploy Notes
Review + Merge on master


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/generic-table-to-storage
```

1. Go to `/data-workflows/table-to-storage/runs` => Listing should be ok
2. Go to `/data-workflows/table-to-storage/runs/:id` => Item informations should be ok
3. Go to `/data-workflows/table-to-storage/configurations` => Listing should be ok
4. Go to `/data-workflows/table-to-storage/configurations/:id` => Item informations should be ok


## Impacted Areas in Application
* `/data-workflows/table-to-storage/*` pages
